cli/cluster.go

```diff
@@ -61,7 +67,9 @@ func runClusterAdd(args *docopt.Args) error {
 		GitHost: args.String["--git-host"],
 		TLSPin:  args.String["--tls-pin"],
 	}
-	if err := config.Add(s); err != nil {
+	f := args.Bool["--force"]
+	d := args.Bool["--default"]
+	if err := config.Add(s, f, d); err != nil {
```

Comment:
Instead of passing the default flag into `config.Add`, I'd call `config.SetDefault` right after calling `config.Add`:

```go
if err := config.Add(s, args.Bool["--force"]); err != nil {
  return err
}
if args.Bool["--default"] {
  if err := config.SetDefault(s.Name); err != nil {
    return err
  }
}
```